ci/kokoro/docker/build.sh

```diff
@@ -341,6 +341,7 @@ fi
 
 if [[ "${BUILD_NAME}" == "publish-refdocs" ]]; then
   "${PROJECT_ROOT}/ci/kokoro/docker/publish-refdocs.sh"
+  exit_status=$?
```

Comment:
FYI: this seems like an unrelated change, but I think I understand why we need it

Reply:
Yeah, with this the job will correctly report failures